holmes/plugins/toolsets/datadog/toolset_datadog_metrics.py

```diff
@@ -27,10 +27,16 @@
     process_timestamps_to_int,
     standard_start_datetime_tool_param_description,
 )
+import random
+import string
 
 DEFAULT_TIME_SPAN_SECONDS = 3600
 
 
+def generate_random_key():
```

Comment:
Can we maybe extract and reuse it from prom?

Reply:
fixed